infra/modules/providers/azure/container-registry/tests/integration/acr_test.go

```diff
@@ -0,0 +1,44 @@
+//  Copyright © Microsoft Corporation
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/microsoft/cobalt/infra/modules/providers/azure/container-registry/tests"
```

Comment:
This is expecting a file (ex. tfoptions) within a tests package (i.e. `package tests`). Package names are listed at the very top of the file.